### PR TITLE
LPS-71463 Add test

### PR DESCRIPTION
**Pull request by leoadb**

### CODE REVIEW

**Comment by leoadb** [comment]
@vicnate5 Could you please help me updating the Poshi? Thanks!

**Comment by vicnate5** [comment]
https://issues.liferay.com/browse/LRQA-31613

**Comment by vicnate5** [comment]
@leoadb I am having trouble compiling this pull.


```
:apps:forms-and-workflow:dynamic-data-mapping:dynamic-data-mapping-data-provider-impl:compileJava
/Users/vicnate5/Liferay/liferay-portal-master/modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/DDMRESTDataProvider.java:74: error: cannot find symbol
                        for (Map<Object, Object> map : ddmDataProviderResponse.getData()) {
                                                                              ^
  symbol:   method getData()
  location: variable ddmDataProviderResponse of type DDMDataProviderResponse
/Users/vicnate5/Liferay/liferay-portal-master/modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/DDMRESTDataProvider.java:144: error: incompatible types: List<Map<Object,Object>> cannot be converted to List<DDMDataProviderResponseOutput>
                return new DDMDataProviderResponse(data);
```

I tried copying over the DDM files to master, running gradlew deploy on DDM
I also tried doing `ant all` on master with the changes here.

Is there another way I can try to compile this?

**Comment by leoadb** [comment]
@vicnate5 Do you have your subrepo branch update with upstream? After run ant-all on liferay-portal you need to run "gradlew -Dbuild.exclude.dirs="dynamic-data-mapping-test,dynamic-data-mapping-test-util" clean deploy" on the root of your subrepo branch. There is code related with this one you mentioned that was not merged to liferay-portal yet. I'm waiting for results for the merge pull request I sent.

**Comment by vicnate5** [comment]
Thanks @leoadb. That worked.

**Comment by vicnate5** [comment]
Sent pull to fix the tests https://github.com/brianchandotcom/liferay-portal/pull/47939

This doesn't need a toggle, so once this is merged to master, we should just run a re-test here.

**Comment by vicnate5** [comment]
ci:retest

**Comment by vicnate5** [comment]
@leoadb This looks good.

**Comment by leoadb** [comment]
@vicnate5 Thanks! @marcellustavares Could you please review this one? Thanks!

File: dynamic-data-mapping-type-validation/src/main/resources/META-INF/resources/validation_field.js [hunk 1]
```diff
@@ -186,6 +184,8 @@ AUI.add(
 
 						var selectedType = instance.get('selectedType');
 
+						debugger;
```
Comment (marcellustavares) [inline dynamic-data-mapping-type-validation/src/main/resources/META-INF/resources/validation_field.js]: @leoadb, please remove this debugger command;

File: dynamic-data-mapping-type-validation/src/main/resources/META-INF/resources/validation_field.js [hunk 2]
```diff
@@ -211,6 +211,8 @@ AUI.add(
 
 						var options = [];
 
+						debugger;
```
Comment (marcellustavares) [inline dynamic-data-mapping-type-validation/src/main/resources/META-INF/resources/validation_field.js]: Same thing.

Comment (leoadb) [inline dynamic-data-mapping-type-validation/src/main/resources/META-INF/resources/validation_field.js]: Final code doesn't have those

**Comment by leoadb** [comment]
@marcellustavares Will resend. I'll also fix the issues I commented on your last pull request. Thanks

**Pull request closed without merging**